### PR TITLE
Update the Rust starter kit

### PR DESCRIPTION
**Pull request by Kerollmops**
Sometimes I follow the api-guideline https://rust-lang-nursery.github.io/api-guidelines/naming.html#c-getter

### CODE REVIEW

File: airesources/Rust/src/hlt/game.rs [hunk 1]
```diff
@@ -41,27 +40,29 @@ impl Game {
 
         println!("{}", name);
```
Comment (Snaar) [inline airesources/Rust/src/hlt/game.rs]: Would you mind attempting to address https://github.com/HaliteChallenge/Halite-II/issues/151 at the same time?

Comment (Kerollmops) [reply airesources/Rust/src/hlt/game.rs]: I will fix that yes, thanks for the remark.

File: airesources/Rust/src/hlt/game.rs [hunk 2]
```diff
@@ -41,27 +40,29 @@ impl Game {
...
 
-        let game = Game {
+        // ignore first line
+        Game::read_line();
```
Comment (Kerollmops) [inline airesources/Rust/src/hlt/game.rs]: Just to know, why did we need to skip the first map update, here ?

Comment (Snaar) [reply airesources/Rust/src/hlt/game.rs]: I think original author didn't understand why we needed it. This original update is what you actually need to implement #151 correctly.

**Comment by Kerollmops** [comment]
The real way to create a game should use [a builder](https://rust-lang-nursery.github.io/api-guidelines/type-safety.html#c-builder) and don't let the user own a complete `Game` type once he hasn't send it's username to the game.

This will avoid the user to send command before sending it's own name.

But this way works fine.

**Pull request merged**